TodoWithDatabase.IntegrationTests/Scenarios/API/Shared/AuthenthicationAndAuthorizationTests.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using TodoWithDatabase.Services;
+using Xunit;
+
+namespace TodoWithDatabase.IntegrationTests.Scenarios.API.Shared
+{
+    [Collection("BaseCollection")]
+    public class AuthenthicationAndAuthorizationTests //: IClassFixture<TestContext> TODO research if this should be in a collection or not
+    {
+        private readonly TestContext _testContext;
+        private readonly string _correctTokenForUser;
+        private readonly string _incorrectToken;
+        private readonly HttpContent _request;
+
+        public AuthenthicationAndAuthorizationTests(TestContext testContext)
+        {
+            _testContext = testContext;
```

Comment:
ennek a sornak mi ertelme? Egyreszt nem hasznalod a _testContext-et, masreszt ez csak referenciat ad at, nem masolja le a peldanyt.
Szolval 
var x = y;
x.a = 2;
Ebben az esetben y.a is 2-lesz. Szoval nincs ertelme uj valtozot letrehozni.